app/views/layouts/application.html.haml

```diff
@@ -15,5 +15,7 @@
         = render 'layouts/header'
       #main.pure-u-1
         - flash.each do |name, msg|
```

Comment:
`name` is being declared in the block and then not used, so it is customary to add an underscore to the front  - as in `_name` to show that this is intentional.